README.md

```diff
@@ -525,6 +525,7 @@ Mac app wrapper around Google's stand-alone Android Messenger. ![javascript_icon
 - [Clean-Me](https://github.com/Kevin-De-Koninck/Clean-Me) - Small macOS app that acts as a system cleaner (logs, cache, ...). ![swift_icon]
 - [Diagnostics](https://github.com/macmade/Diagnostics) - Diagnostics is an application displaying the diagnostic reports from applications on macOS. ![swift_icon]
 - [DisableMonitor](https://github.com/Eun/DisableMonitor) - Easily disable or enable a monitor on your Mac. ![objective_c_icon]
+- [Disk Inventory X](http://www.derlien.com/) - Shows the sizes of files and folders in a special graphical way called "treemaps". ![objective_c_icon]
```

Comment:
HI @dutzi, can you change website URL to git URL?

Reply:
I wish, turns out it's not open source 😕

Reply:
I'm sorry but i can't merge it( I'll close this PR.